cmd/juju/action/run.go

```diff
@@ -56,9 +56,10 @@ use the --background option.
 
 To set the maximum time to wait for a action to complete, use the --wait option.
 
-By default, the output of a single action will just be that action's stdout.
-For multiple actions, each action stdout is printed with the action id.
-To see more detailed information about run timings etc, use --format yaml.
+By default, a single action will output it's failure message if the action fails,
+followed by any results set by the action. For multiple actions, each action's
```

Comment:
should be ...`any result set`... right?

Reply:
Multiple results can be set (we print a list of key-value pairs)